Makefile

```diff
@@ -45,20 +46,35 @@ DELETE_STACK=aws cloudformation delete-stack --stack-name
 
 $(shell mkdir -p ${BUILD_DIR_PATH} && touch ${BUILD_DIR_PATH}/_go.mod)
 
+version:
+	@echo ${VERSION}
+
+repo-full-name:
+	@echo ${REPO_FULL_NAME}
+
+bin-name:
+	@echo ${BIN}
+
+latest-release-tag:
+	@echo ${LATEST_RELEASE_TAG}
```

Comment:
can we align how we calculate tags with [NTH](https://github.com/aws/aws-node-termination-handler/blob/main/Makefile#L2)